t/xrpc/redis.t

```diff
@@ -565,3 +577,126 @@ passed
 --- response_body
 ok
 --- stream_conf_enable
+
+
+
+=== TEST 12: set custom log format
```

Comment:
Maybe we can split the redis variable into another PR and the test can be put into `redis2.t`. Otherwise, there will be merge conflict.

Reply:
split this PR, so close.